src/TorchSharp/LinearAlgebra.cs

```diff
@@ -518,10 +518,11 @@ public static (Tensor Q, Tensor R) qr(Tensor input, QRMode mode = QRMode.Reduced
             /// </summary>
             /// <param name="input">Tensor of shape (*, n, n) where * is zero or more batch dimensions.</param>
             /// <param name="other">Right-hand side tensor of shape (*, n) or (*, n, k) or (n,) or (n, k)</param>
+            /// <param name="left">whether to solve the system AX = B or XA = B.</param>
             /// <returns></returns>
-            public static Tensor solve(Tensor input, Tensor other)
+            public static Tensor solve(Tensor input, Tensor other, bool left = true)
```

Comment:
You mentioned in the release notes there is no API changes or bugs fixes. technically this is not breaking but still it can break if anyone used reflection to bind to such method which I think very unlikely.

Reply:
Thank you for catching that, I forgot that I had made some minor changes in accordance with change PT APIs.